client/src/components/kvEditor/KVEditor.tsx

```diff
@@ -9,14 +9,23 @@ type KVEditorProps = {
   setKvs?: any;
   name: string;
   readOnly?: boolean;
+  canDisableRows?: boolean;
```

Comment:
we can make it non-optional but with default = false

Reply:
Making it in non-optional + defining the default value in `function KVEditor()` gives ts-errors in the parent components for KVEditor instances where we don't pass canDisableRows, because it is non-optional.
This error appears eg. in the RequestPanel for the "param"-instance of the KVEditor.

Reply:
true, thought it would work the same way as in regular javascript.